### ISSUE TITLE
503 Slowdown S3 error

### ISSUE THREAD

**Issue by theromie** [workfall/multipart-upload-backend#1]
Hi, I am following your example to upload a large file to S3. But I am getting 503 Slowdown S3 error. How can I slow down the rate limit?

**Comment by workfall** [comment]
@theromie you might want to do away with the expiry param ~~Expires: 60~~ for the pre-signed URL
```typescript
async getPreSignedUrl(body) {
    const params = {
      Bucket: this.AWS_S3_BUCKET,
      Key: String(body.name),
      //   Expires: 60, 
      ContentType: body.contentType,
    };

    return await this.s3.getSignedUrlPromise('putObject', params);
  }
```

**Comment by workfall** [comment]
@theromie we are going to create a PR for that. If it works for you, please let us know so that we close this issue.

**Comment by theromie** [comment]
Nope. It's not working. I am uploading files of around 450MB and dividing them into 50MB chunks. Out of 9 connection, 3-4 active connection is failing with 503. Do you have any idea how to handle the failed connection to retry?

**Comment by workfall** [comment]
We are looking for an appropriate CloudWatch config solution, hold on please.

**Comment by theromie** [comment]
Hey! I am able to solve this by looking into this solution. 
[logrocket](https://blog.logrocket.com/multipart-uploads-s3-node-js-react/)
A small tweak is done in this blog.
